src/SingleSelect/Input.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import propTypes from '@dhis2/prop-types'
 import cx from 'classnames'
+import requiredIf from 'react-required-if'
```

Comment:
Missed one here.

Reply:
Yeah I was incrementally updating and force pushing, should be all good now.

Reply:
Looks good to me!